Makefile

```diff
@@ -26,6 +26,12 @@ lint-ci: download
 	make lint
 test: build
 	bazel test $(BAZEL_BUILD_OPTS) --test_output=all //...
+test-mac:
```

Comment:
Please add a comment just above this saying that this make target is a workaround for https://github.com/bazelbuild/rules_go/issues/2013